### PR TITLE
[Routing] Add doc about unicode requirements

### PR DESCRIPTION
**Pull request by nicolas-grekas**
Ref. https://github.com/symfony/symfony/pull/19604

### CODE REVIEW

File: routing/requirements.rst [hunk 1]
```diff
@@ -159,6 +159,14 @@ Path     Parameters
...
 =======  ========================
 
+.. note::
+
+    Since Symfony 3.2, requirements can be prefixed by the ``*`` character to
+    enable Unicode matching, thus making ``.`` match any UTF-8 character instead
+    of just a single byte. Unicode matching is also automatically enabled
+    whenever the route or a requirement contains a non-ASCII UTF-8 character or
+    a `PCRE Unicode property`_ (``\p{xx}``, ``\P{xx}`` or ``\X``).
+
```
Comment (fabpot) [inline routing/requirements.rst]: I think we need an example with the `*` prefix to make things real clear.

**Comment by nicolas-grekas** [comment]
Doc synced with the PR (unicode parameter instead of prefix)

**Comment by javiereguiluz** [comment]
👍 

Status: reviewed

---

A fantastic feature and perfectly documented! Thanks @nicolas-grekas 

(Note: there are some minor formatting issues in this commit: in RST, code literals must be wrapped with double backticks instead of Markdown's single backticks).

**Comment by xabbuh** [comment]
Thank you @nicolas-grekas.

**Pull request merged**